script.js

```diff
@@ -1,139 +1,249 @@
 
-
+//exercise to translate words
+//VARIABLES
+// counter trough items(words)
 let countItem =0;
+// id input from user in inputfield 
 let inputValue = document.getElementById("inputvalue");
```

Comment:
`input` or `inputElement` or `inputField` is a better naming since you are not getting the value but a reference to the field/element